notebooks/classification/mcmc_bnn_makemoons.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    logits = model.apply(one_state, X)
</pre><p>code here and in the below cell is largely same. Can we use some simple utilities and avoid code duplication?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nipunbatra/bnn-zoo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>